mkdocs.yml

```diff
@@ -45,16 +46,16 @@ plugins:
   - search
   - gallery:
         conf_script: docs/gallery_conf.py
-        # path to the directory containing our background/neural_modeling
+        # path to the directory containing our background/tutorials
```

Comment:
this comment really isn't adding much

the difference between `examples_dirs` and `gallery_dirs` is that one is the input and one is the output of mkdocs-gallery, right? so that would be more helpful comments